### PR TITLE
Adding new objects and attributes for Security Finding: Proposal #490

### PR DESCRIPTION
**Pull request by pagbabian-splunk**
From Proposal #490 adding objects and attributes from the Splunk Detection Report extension to Findings, to the core Security Finding event class.  Note, these are the objects.  There is another Proposal for other attributes, a change in an activity_id caption, and ultimately an update to the Security Finding event class still to come.

### CODE REVIEW

**Comment by pagbabian-splunk** [comment]
Also added attributes for Security Finding from Proposal #491 including a new `impact_score` integer attribute and changing the Splunk extension type of `impact` to a `string_t` sibling.

**Review by irakledibm — DISMISSED** [review]

**Review by floydtree — CHANGES_REQUESTED** [review]

File: objects/kill_chain.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "caption": "Kill Chain",
+    "description": "The <a target='_blank' href='https://www.lockheedmartin.com/en-us/capabilities/cyber/cyber-kill-chain.html'>Cyber Kill Chain®</a>.",
+    "extends": "object",
+    "name": "kill_chain",
+    "attributes": {
+      "phase": {
+        "requirement": "recommended"
+          },
+      "phase_id": {
```
[the remark this answers is not in the document]
Comment (pagbabian-splunk) [reply objects/kill_chain.json:10]: Ah, missed that one.  Thanks.  Will update.

**Review by rroupski — APPROVED** [review]

**Review by irakledibm — DISMISSED** [review]
looks good

File: events/findings/findings.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "activity_id": {
       "enum": {
         "1": {
-          "caption": "Generate",
+          "caption": "Create",
           "description": "A security finding is generated."
```
Comment (floydtree) [inline events/findings/findings.json:12]: Might as well update the description to reflect the change?

**Review by floydtree — CHANGES_REQUESTED** [review]

File: events/findings/findings.json (hunk 2 above)
Comment (pagbabian-splunk) [reply events/findings/findings.json]: Yes, just pushed that change.

**Review by floydtree — APPROVED** [review]

**Review by rroupski — APPROVED** [review]

**Review by rroupski — APPROVED** [review]

**Review by irakledibm — APPROVED** [review]

**Pull request merged**